src/Components/test/E2ETest/ServerExecutionTests/InteropReliabilityTests.cs

```diff
@@ -439,13 +439,13 @@ public async Task DispatchingEventsWithInvalidEventHandlerId()
 
             await Client.HubConnection.InvokeAsync(
                 "DispatchBrowserEvent",
-                JsonSerializer.Serialize(browserDescriptor, new JsonSerializerOptions() { PropertyNamingPolicy = JsonNamingPolicy.CamelCase }),
-                JsonSerializer.Serialize(mouseEventArgs, new JsonSerializerOptions() { PropertyNamingPolicy = JsonNamingPolicy.CamelCase }));
+                JsonSerializer.Serialize(browserDescriptor, TestJsonSerializerOptionsProvider.Options),
+                JsonSerializer.Serialize(mouseEventArgs, TestJsonSerializerOptionsProvider.Options));
 
             Assert.Contains(
                 logEvents,
                 e => e.eventIdName == "DispatchEventFailedToDispatchEvent" && e.logLevel == LogLevel.Debug &&
-                     e.exception is ArgumentException ae && ae.Message.Contains("There is no event handler with ID -1"));
+                     e.exception is ArgumentException ae && ae.Message.Contains("There is no event handler with ID 1"));
```

Comment:
@BrennanConroy changed the event handler id in the dependency update (they're unsigned now and it previously had a negative value). Fixing it up. Also using the options provider

Reply:
Haha, one day these tests will be runnable